durable-storage/src/repo.rs

```diff
@@ -57,4 +76,9 @@ impl DirectoryManager {
 
         Ok(tempdir)
     }
+
+    /// Obtain the path to the commit directory for the given `id`.
+    pub fn commit_dir(&self, id: &CommitId) -> PathBuf {
+        self.commits_dir.join(id.to_bs58())
```

Comment:
What is the reason for picking base 58? Is it better than hex encoding, for example?